fineract-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/domain/LoanProduct.java

```diff
@@ -385,6 +391,11 @@ public static LoanProduct assembleFromJson(final Fund fund, final String loanTra
         final Integer overDueDaysForRepaymentEvent = command
                 .integerValueOfParameterNamed(LoanProductConstants.OVER_DUE_DAYS_FOR_REPAYMENT_EVENT);
 
+        final boolean enableDownPayment = command.parameterExists(LoanProductConstants.ENABLE_DOWN_PAYMENT)
+                && command.booleanPrimitiveValueOfParameterNamed(LoanProductConstants.ENABLE_DOWN_PAYMENT);
+        final BigDecimal disbursedAmountPercentageDownPayment = command
```

Comment:
This should be checked and fetch if the down payment is enabled, but in that case it is required and must be greater than 0.